db/connection.js

```diff
@@ -1,12 +1,18 @@
 const { Pool } = require('pg');
 const ENV = process.env.NODE_ENV || 'development';
+const config = {};
 
 require('dotenv').config({
   path: `${__dirname}/../.env.${ENV}`,
 });
 
-if (!process.env.PGDATABASE) {
-  throw new Error('PGDATABASE not set');
+if (ENV === 'production') {
+  config.connectionString = process.env.DATABASE_URL;
+  config.max = 2;
+}
+
+if (!process.env.PGDATABASE && !process.env.DATABASE_URL) {
+  throw new Error('PGDATABASE or DATABASE_URL not set');
 }
 
 module.exports = new Pool();
```

Comment:
I have passed **config** to **Pool** as an argument in the following push 
`module.exports = new Pool(config);`